packages/blitz/src/cli/utils/check-latest-version.ts

```diff
@@ -0,0 +1,185 @@
+import findUp from "find-up"
+import resolveFrom from "resolve-from"
+import {join, dirname} from "path"
+import fs from "fs"
+import {readVersions, resolveVersionType} from "./read-versions"
+import {getPkgManager} from "./helpers"
+import {log} from "../../logging"
+
+const BLITZ_ENDPOINT = `https://registry.npmjs.org/-/package/blitz/dist-tags?foo=bar`
```

Comment:
We don't need the foo=bar? (it returns the same thing for me with/without?)
Also maybe have a `BASE_NPM_URL` and just add package to it to make it harder to add typos here